addons/l10n_in_withholding_payment/models/account_payment.py

```diff
@@ -0,0 +1,21 @@
+from odoo import models, fields
+
+
+class AccountPayment(models.Model):
+    _inherit = "account.payment"
+
+    l10n_in_withhold_payment_move_ids = fields.One2many(
+        'account.move', 'l10n_in_withholding_ref_payment_id',
+        string="Indian Payment TDS Entries"
+    )
+    l10n_in_total_withholding_payment_amount = fields.Float(compute='_compute_l10n_in_total_withholding_payment_amount')
```

Comment:
```suggestion
    l10n_in_withhold_payment_move_ids = fields.One2many(
        'account.move', 'l10n_in_withholding_ref_id',
        string="Indian Payment TDS Entries"
    )
    l10n_in_total_withholding_amount = fields.Float(compute='_compute_l10n_in_total_withholding_payment_amount')
```
That way, it overrides the definitions in the other module and we do not have the confusion whether the link is with the payment or with the payment's journal entry.  It would be always with the payment.